src/utils.cairo

```diff
@@ -133,3 +133,26 @@ pub fn double_sha256(a: u256, b: u256) -> u256 {
         low: x4.into() * TWO_POW_96 + x5.into() * TWO_POW_64 + x6.into() * TWO_POW_32 + x7.into(),
     }
 }
+
```

Comment:
copy pasted these utils that were in the `Tests` crate, couldnt find a way to import them and use them in ./scr/... files